charts/aws-ebs-csi-driver/templates/csidriver.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: storage.k8s.io/v1
+apiVersion: storage.k8s.io/v1beta1
```

Comment:
As v1beta1 will be deprecated soon(likely 3 months), can you update the apiVersion based on kubeVersion? something like
`{{ ternary "storage.k8s.io/v1" "storage.k8s.io/v1beta1" (semverCompare ">=1.19.0-0" .Capabilities.KubeVersion.Version) }}`
Also, since this is helm chart change, can you also bump the chart version to 1.0.2?